data/Alexxanderr20.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Edward Munoz" 
+  "bio": "Aspiring software engineer who is focused on a front-end tech stack, but is open to any kind of     software stack to learn and grow." ,
```

```suggestion
  "bio": "Aspiring software engineer who is focused on a front-end tech stack, but is open to any kind of software stack to learn and grow.",
```

extra white spaces can be removed for consistency throughout the repository :)